library_generation/utils/utilities.sh

```diff
@@ -158,7 +158,6 @@ download_tools() {
   local grpc_version=$3
   local os_architecture=$4
   pushd "${output_folder}"
-  download_generator_artifact "${gapic_generator_version}" "gapic-generator-java-${gapic_generator_version}.jar"
```

Comment:
Can you please update the comment above as well?

Reply:
Thank you, done.